spec/factories/work_factory.rb

```diff
@@ -150,7 +150,8 @@
       additional_credit {
         [
           Work::AdditionalCredit.new({"name"=>"Douglas Lockard", "role"=>"photographed_by"}),
-          Work::AdditionalCredit.new({"name"=>"Mark Backrath",   "role"=>"photographed_by"})
+          Work::AdditionalCredit.new({"name"=>"Mark Backrath",   "role"=>"photographed_by"}),
+          Work::AdditionalCredit.new({"name"=>"Jocelyn R. McDaniel",   "role"=>"translator"})
```

Comment:
Why are you adding this into default "work factory", what were your thoughts/motivation there?

Reply:
Hmmmm.....  so supposing, in the *existing* tests, there's some weird interaction that causes problems if you add a translator or have more than one "role" in `additional_credit`, etc., this would make us aware of it.

Secondarily, if a bug involving `translator` comes up _later_, then this metadata is there in all the existing tests that use a `Work` object, and we have it to test with.

Reply:
OK, maybe for #1. 

For #2, it's importnat to keep in mind we don't create a generic factory with the data to exersize every possible test we'd want to do -- we create specific data in specific tests for the test. If we want to write a test to (eg) make sure the translator is displayed properly (as below is suggested we might), we'd want to actually write `create(:work, additional_credit: whatever)`, not just rely on whatever is in the default factory. Relying on what's in the default factory makes it too fragile to default factory changes, the default factory should just be a "typical default generic thing". 

Because *most* of our Works won't have translators is what makes me think the default factory shoudln't. (Yes, this could apply to photographer too, it's true). Plus everything we add to the default factory making it heavier-weight is potentially making every single test that uses it slightly slower, adding up. 

Not a big deal either way, just my thoughts.

Reply:
OK you convinced me again.